docs/designs/crd.md

```diff
@@ -492,6 +501,34 @@ Every Keystore Value defines a secret to pull secrets from.
             <th>default</th>
         </tr>
     </thead>
+    <tbody><tr>
+        <td><b>enable</b></td>
+        <td>bool</td>
+        <td>Define if to enable monitoring for that cluster</td>
+        <td>true</td>
+        <td>-</td>
+      </tr><tr>
+        <td><b>monitoringUserSecret</b></td>
+        <td>[]string</td>
+        <td>Define from which user the monitor will run (Getting Secret name, the secret should contain 'username':'password' fileds).</td>
+        <td>false</td>
+        <td>admin</td>
+      </tr><tr>
+        <td><b>interval</b></td>
+        <td>string</td>
+        <td>Define interval for scraping</td>
+        <td>false</td>
+        <td>30s</td>
+      </tr><tr>
+      </tr><tr>
+        <td><b>PluginURL</b></td>
+        <td>string</td>
+        <td>Define offline link to Aiven Plugin</td>
+        <td>false</td>
+        <td>https://github.com/aiven/prometheus-exporter-plugin-for-opensearch</td>
```

Comment:
The example URL should be a complete path with versions and all as the operator expects it. Also missing is info that the plugin must match the opensearch version.

Reply:
Cause that i dont want to give an example that related to 1 specific version, i gave the plugin repo.
The operator knows how to build the plugin with the needed version automatically

Reply:
But thats the point, if you supply a custom url the operator does not know how to build a complete URL with versions, it just takes the URL as-is. So giving just the base path will lead to errors. You should give an URL with placeholders for the opensearch version so the users know what to put in.

Reply:
ho ok, np, added <YOUR_CLUSTER_VERSION> to the example